config/default_settings.yaml

```diff
@@ -0,0 +1,17 @@
+# Default arguments used for the CLI. Can be overridden using the corresponding
+# flag at runtime
+default_args:
+  run_github_workflow: True
+  run_glue_crawler: True
+  upload_logs: True
+  upload_data: True
```

Comment:
I defined these here because it was nice to be able to quickly change runtime flags here instead of in the CLI. Open to removing them if this is too confusing/weird!

Reply:
**[Praise]** I like this! In my experience, it's a common pattern for CLIs to allow for overriding values using a config file.